### PR TITLE
add tests for deleting reference without permissions

### PR DESCRIPTION
**Pull request by juhanikat**

### CODE REVIEW

File: backend/src/api-tests/reference/delete.test.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import { login, logout, resetDatabase, resetDatabaseTimeout, send } from '../utils'
+
+describe('Deleting a locality works', () => {
+  beforeAll(async () => {
+    await resetDatabase()
+  }, resetDatabaseTimeout)
+  beforeEach(async () => {
+    await login()
+  })
+
```
Comment (iritmaximus) [inline backend/src/api-tests/reference/delete.test.js:11]: This is missing a `pool.end()`. See https://github.com/nowcommunity/nowdatabase/blob/5ef2c82582e32d8166d972b5753c3a4a36bd60e6/backend/src/api-tests/reference/create.test.ts#L6-L19

**Review by iritmaximus — CHANGES_REQUESTED** [review]

**Review by iritmaximus — CHANGES_REQUESTED** [review]
I am very sorry about these multiple changes. The file should be `.ts`. I didn't notice it previously.

**Comment by juhanikat** [comment]
Renamed :+1:

**Review by iritmaximus — CHANGES_REQUESTED** [review]
I seriously hate to be this person, but you didn't remove the `.js` file :D

**Comment by juhanikat** [comment]
made a new pull request for this #422

**Pull request closed without merging**